### DISCUSSION TITLE
Wrong SSID PAssword

### DISCUSSION THREAD

**Discussion by highwingers** [davesteele/comitup#188] [Q&A]
is there anyway to let the user know that they entered a "wrong" wifi password?

**Comment by davesteele** [comment]
If authentication with the upstream AP fails, Comitup will re-create the "comitup" hotspot.

**Reply by highwingers** [reply]
anyway to change the time from 3 minutes to 1 minute?

**Comment by davesteele** [comment]
Are you asking about two different things? If a connection fails, the failure takes however long NetworkManager uses to make that determination, which should be a lot less than 3 minutes. Once the failure returns to comitup, it will restart the hotspot.

Comitup will normally make another connection attempt in 3 minutes, if no-one is connected to the hotspot (otherwise, that user is in control). That timeout is [hard coded](https://github.com/davesteele/comitup/blob/6554569db00eea121082305bb3f1c50aa743a1c4/comitup/states.py#L327).

However if the failure is an authentication error, the connection information is deleted, and there are no timeouts or retries for that connection. The was added in [May](https://github.com/davesteele/comitup/issues/106) (version 1.24).
